src/inference/mh.jl

```diff
@@ -1,173 +1,308 @@
-mutable struct MHState{V<:VarInfo} <: AbstractSamplerState
-    proposal_ratio        ::   Float64
-    prior_prob            ::   Float64
-    violating_support     ::   Bool
-    vi                    ::   V
+import AdvancedMH
+const AMH = AdvancedMH
+
+import DynamicPPL: VarName, _getranges, _getindex, getval, _getvns
```

Comment:
Maybe move the import statements to `Inference.jl`?

Reply:
Sure.